.gitattributes

```diff
@@ -0,0 +1 @@
+*.png binary
```

Comment:
This can be removed, there are no `png` files in the repository.

Reply:
Will be when #3 will be done as next PR